modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -8,12 +8,18 @@ module Veterans
       class IntentToFileController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
-        ITF_TYPES = %w[compensation pension burial].freeze
-
+        # GET to fetch active intent to file by type
+        #
+        # @return [JSON] ITF record
         def type
+          itf_types = ClaimsApi::V2::ParamsValidation::IntentToFile::IntentToFileValidator.itf_types
```

Comment:
same as this comment... same add though, so makes sense: https://github.com/department-of-veterans-affairs/vets-api/pull/11411/files#r1056534435